tests/test_array/good/aggregation_2.catala_en

```diff
@@ -33,13 +33,14 @@ scope B:
 $ catala Interpret -s A
 [RESULT] Computation successful! Results:
 [RESULT] x =
-           [S { "id"= 0; "income"= $0.00 }; S { "id"= 1; "income"= $9.00 };
-             S { "id"= 2; "income"= $5.20 }]
+           [ {S "id" = 0; "income" = $0.00};
+             {S "id" = 1; "income" = $9.00};
+             {S "id" = 2; "income" = $5.20} ]
 ```
 
 ```catala-test-inline
 $ catala Interpret -s B
 [RESULT] Computation successful! Results:
-[RESULT] argmax = S { "id"= 1; "income"= $9.00 }
-[RESULT] argmin = S { "id"= 0; "income"= $0.00 }
+[RESULT] argmax = {S "id" = 1; "income" = $9.00}
```

Comment:
This printing behavior is a bit weird. The previous one was influenced by Rust's `struct`. Maybe `S.{ ... }` ?

Reply:
Yeah this was inspired by OCaml's `{ Foo.bar = _; baz = _ }` but in that case `Foo` is a module; I have no strong opinion on this, whatever is more readable and understandable.

Reply:
@adelaett has another take in #412, `{ ...}_S`